CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Changelog
 
+## [1.2](https://github.com/arborx/arborx/tree/v1.2) (2022-04-01)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.1...v1.2)
+
+**New features:**
+
+- Implement Euclidean minimum spanning tree algorithm [\#624](https://github.com/arborx/ArborX/pull/624), [\#631](https://github.com/arborx/ArborX/pull/631), [\#661](https://github.com/arborx/ArborX/pull/631)
+
+**Build changes:**
+
+- Require Kokkos 3.4 [\#578](https://github.com/arborx/ArborX/pull/578)
+
+**Enhancements:**
+
+- Indicate the status of the worktree in the hash string ("-dirty") [\#558](https://github.com/arborx/ArborX/pull/558)
+- Check valid callback and space accessibility [\#563](https://github.com/arborx/ArborX/pull/563)
+- Add a molecular dynamics example [\#564](https://github.com/arborx/ArborX/pull/564)
+- Add nightly testing against Kokkos develop [\#569](https://github.com/arborx/ArborX/pull/569)
+- Improve performance of the HIP backend when using HIP-Clang [\#575](https://github.com/arborx/ArborX/pull/575)
+- Remove PGI CI [\#590](https://github.com/arborx/ArborX/pull/590)
+- Add Point-Sphere intersection algorithm [\#584](https://github.com/arborx/ArborX/pull/584)
+- Avoid host copy in distributed tree construction when using CUDA-aware MPI [\#597](https://github.com/arborx/ArborX/pull/597)
+- Print Kokkos version during configuration [\#609](https://github.com/arborx/ArborX/pull/609)
+- Improve performance of the brute force algorithm [\#616](https://github.com/arborx/ArborX/pull/616)
+- Use 64-bit Morton indices by default in the construction [\#637](https://github.com/arborx/ArborX/pull/637)
+- Allow alternate space filling curves (EXPERIMENTAL) [\#646](https://github.com/arborx/ArborX/pull/646)
+- Remove certain helper functions from the `ArborX` namespace [\#648](https://github.com/arborx/ArborX/pull/648)
```

Comment:
This falls under deprecation, not enhancement

Reply:
It's also pretty vague and thus doesn't sound significant.

Reply:
Clarified and moved to deprecated.